src/radio/index.js

```diff
@@ -148,6 +151,16 @@ class Radio extends EventEmitter {
     this.emit('skips', this.skips, needed);
   }
 
+  async addSongWrapper(link: string, user: Discord.User) {
+    const passedUrl = new URL(link);
+    if (passedUrl.pathname === '/playlist') {
+      return playlistParser(String(passedUrl.searchParams.get('list'))).then(items =>
+        items.map(item => this.addSong(item, user))
+      );
+    } else {
```

Comment:
we don't need to put this under an `else`
(removing the else will make it easier to follow in case we add more playlist detection and handling)